gatsby-node.js

```diff
@@ -7,7 +7,8 @@
 // You can delete this file if you're not using it
 const path = require('path');
 const _ = require('lodash');
-
+const themeRoot = require('./theme-path');
```

Comment:
Get the path of from where the components to be imported from. `theme-path` uses the config file to do that.